src/expr/src/interpret.rs

```diff
@@ -662,9 +641,9 @@ impl Interpreter for Trace {
             return Pushdownable::No;
         }
 
-        let is_monotonic = variadic_monotonic(func);
+        let is_monotone = func.is_monotone();
         exprs.into_iter().fold(Pushdownable::No, |acc, arg| {
-            acc.max(arg.min(is_monotonic.into()))
+            acc.max(arg.min(is_monotone.into()))
         })
```

Comment:
Done! To your question:
- Max is join; min is meet.
- Consider some arbitrary variadic function call `f(a, ...)`. We say that this expression is "pushdownable" if both:
  - Any of the arguments are pushdownable. (eg. a column reference, or another pushdownable expression.)
  - The function itself is monotonic / similar.

This will hopefully get simpler once I get rid of the `Maybe` case and some of the dedicated types in the followup PR, as you suggested above...

Reply:
If it helps, I think of "monotone" as the nice crisply-defined mathematical concept and "pushdownable" as the muddier idea that captures more or less what we're able to calculate interesting range information for in practice.